diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.4.0/createlinks mezzanine_patched_e-smith-imap-1.4.0/createlinks
--- e-smith-imap-1.4.0/createlinks	2006-03-14 13:19:01.000000000 -0500
+++ mezzanine_patched_e-smith-imap-1.4.0/createlinks	2006-03-24 16:28:42.000000000 -0500
@@ -30,7 +30,10 @@
     network-delete
     ))
 {
-    safe_symlink("sigusr1", "root/etc/e-smith/events/$event/services2adjust/imap");
+    safe_symlink("sigusr1",
+	"root/etc/e-smith/events/$event/services2adjust/imap");
+    safe_symlink("sigusr1",
+	"root/etc/e-smith/events/$event/services2adjust/imaps");
 }
 
 foreach my $event (qw(post-upgrade))
