diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imap/ConcurrencyLimitPerIP mezzanine_patched_e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imap/ConcurrencyLimitPerIP
--- e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imap/ConcurrencyLimitPerIP	2006-03-15 05:19:01.000000000 +1100
+++ mezzanine_patched_e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imap/ConcurrencyLimitPerIP	2006-04-07 12:18:40.035393900 +1000
@@ -1 +1 @@
-6
+12
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imaps/ConcurrencyLimitPerIP mezzanine_patched_e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imaps/ConcurrencyLimitPerIP
--- e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imaps/ConcurrencyLimitPerIP	2006-03-15 05:19:01.000000000 +1100
+++ mezzanine_patched_e-smith-imap-1.4.0/root/etc/e-smith/db/configuration/defaults/imaps/ConcurrencyLimitPerIP	2006-04-07 12:18:54.605872896 +1000
@@ -1 +1 @@
-6
+12
