diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/access mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/access
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/access	2003-03-21 09:00:39.000000000 -0700
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/access	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-private
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/status mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/status
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/status	2003-03-21 09:00:39.000000000 -0700
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/status	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/TCPPort mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/TCPPort
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/TCPPort	2005-08-10 15:34:34.707194904 -0600
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/TCPPort	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-143
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/type mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/type
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/type	2003-03-21 09:00:39.000000000 -0700
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imap/type	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-service
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/access mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/access
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/access	2005-08-10 15:34:34.657202504 -0600
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/access	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-private
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/status mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/status
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/status	2005-08-10 15:34:34.709194600 -0600
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/status	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/TCPPort mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/TCPPort
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/TCPPort	2005-08-10 15:34:34.708194752 -0600
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/TCPPort	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-993
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/type mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/type
--- e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/type	2005-08-10 15:34:34.661201896 -0600
+++ mezzanine_patched_e-smith-imap-1.3.1/root/etc/e-smith/db/configuration/defaults/imaps/type	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-service
