diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.4.0/root/var/service/imap/run mezzanine_patched_e-smith-imap-1.4.0/root/var/service/imap/run
--- e-smith-imap-1.4.0/root/var/service/imap/run	2006-03-14 13:19:01.000000000 -0500
+++ mezzanine_patched_e-smith-imap-1.4.0/root/var/service/imap/run	2006-03-19 13:40:50.000000000 -0500
@@ -14,7 +14,7 @@
     -v \
     -i ./peers \
     -c ${INSTANCES:-400} \
-    -C ${PER_IP_INSTANCES:-6}:'421 per host concurrency limit reached\r\n' \
+    -C ${INSTANCES_PER_IP:-6}:'421 per host concurrency limit reached\r\n' \
     -l ${LOCALNAME:-0} \
     ${LISTENIP:-0} \
     ${PORT:-imap} \
diff -Nur -x '*.orig' -x '*.rej' e-smith-imap-1.4.0/root/var/service/imaps/run mezzanine_patched_e-smith-imap-1.4.0/root/var/service/imaps/run
--- e-smith-imap-1.4.0/root/var/service/imaps/run	2006-03-14 13:19:01.000000000 -0500
+++ mezzanine_patched_e-smith-imap-1.4.0/root/var/service/imaps/run	2006-03-19 13:41:09.000000000 -0500
@@ -8,7 +8,7 @@
 	-v \
 	-i ./peers \
 	-c ${INSTANCES:-400} \
-	-C ${PER_IP_INSTANCES:-6}:'421 per host concurrency limit reached\r\n' \
+	-C ${INSTANCES_PER_IP:-6}:'421 per host concurrency limit reached\r\n' \
 	-l ${LOCALNAME:-0} \
 	${LISTENIP:-0} \
 	${PORT:-imaps} \
